features/IFC102_Absence-of-deprecated-entities.feature

```diff
@@ -1,74 +1,319 @@
+      | IfcWindowLiningProperties     | ShapeAspectStyle              |
+      | IfcTendon                     | SteelGrade                    |
+      | IfcTendon                     | NominalDiameter               |
+      | IfcTendon                     | CrossSectionArea              |
+      | IfcTendon                     | TensionForce                  |
+      | IfcTendon                     | PreStress                     |
+      | IfcTendon                     | FrictionCoefficient           |
+      | IfcTendon                     | AnchorageSlip                 |
+      | IfcTendon                     | MinCurvatureRadius            |
+      | IfcTendonArchor               | SteelGrade                    |
+      | IfcTendonType                 | NominalDiameter               |
+      | IfcTendonType                 | CrossSectionArea              |
+      | IfcTendonType                 | SheathDiameter                |
+
+
+  Scenario Outline: Check for deprecated attributes - IFC2X3  
+
+      Given A model with Schema "IFC2X3"
+      Given an <Entity>
+
+      Then <Attribute> = empty
+
+    Examples:
+      | Entity                        | Attribute                     |
+      | IfcFillAreaStyleHatching      | PointOfReferenceHatchLine     |
+
+
+  Scenario Outline: Check for deprecated enumerated values - IFC4.3
+    # IfcBuildingElementProxy: IfcBuildingElementProxyTypeEnum
+    # IfcFireSuppressionTerminal: IfcFireSuppressionTerminalTypeEnum
+    # IfcCableCarrierFitting: IfcCableCarrierFittingTypeEnum
+    # IfcGeographicElement: IfcGeographicElementTypeEnum
+    # IfcSpace: IfcSpaceTypeEnum
+    # IfcWall: IfcWallTypeEnum
+
+    Given A model with Schema "IFC4.3"
+    Given an <Entity>
+
+    Then PredefinedType is not <Value>
+
+    Examples:
+      | Entity                          |        Value                   |
+      | IfcBuildingElementProxy         | "PROVISIONFORSPACE"            |
+      | IfcBuildingElementProxy         | "PROVISIONFORVOID"             |
+      | IfcFireSuppressionTerminal      | "SPRINKLERDEFLECTOR"           |
+      | IfcFireSuppressionTerminalType  | "SPRINKLERDEFLECTOR"           |
+      | IfcCableCarrierFitting          | "TEE" or "CROSS" or "REDUCER"  |
+      | IfcCableCarrierFittingType      | "TEE" or "CROSS" or "REDUCER"  |
+      | IfcGeographicElement            | "SOIL_BORING_POINT"            |
+      | IfcGeographicElementType        | "SOIL_BORING_POINT"            |
+      | IfcSpace                        | "INTERNAL" or "EXTERNAL"       |
+      | IfcSpaceType                    | "INTERNAL" or "EXTERNAL"       |
+      | IfcStructuralLoadGroup          | "LOAD_COMBINATION"             |
+      | IfcWall                         | "POLYGONAL" or "STANDARD"      |
+      | IfcWallType                     | "POLYGONAL" or "STANDARD"      |
+
+
+  Scenario Outline: Check for deprecated enumerated values - IFC4
+    # IfcStructuralLoadGroup: IfcLoadGroupTypeEnum
```

Comment:
Similar comment; looks like the comments need to be removed.

Reply:
[Updated](https://github.com/buildingSMART/ifc-gherkin-rules/pull/223/commits/2292cd7555984e6f4e5ab0a706dea3907edcd09e)